### DISCUSSION TITLE
2.3 Alert screen refresh - automatic?

### DISCUSSION THREAD

**Discussion by rosswakelin** [Security-Onion-Solutions/securityonion#11005] [Unsupported Versions] [answered]
We would like our alert screen to refresh automatically, without have to go into the Options dropdown and set the refresh interval.  Is there a hidden setting somewhere that will set a default refresh period?

**Comment by dougburks** [comment, marked as answer]
This feature was added to 2.4:

FEATURE: Add relative Timeframe and Refresh Interval as URL Parameters
https://github.com/Security-Onion-Solutions/securityonion/issues/3352
